src/runtime/instance.rs

```diff
@@ -37,7 +37,8 @@ pub enum InstanceABI {
 }
 
 impl Instance {
-    pub(in crate::runtime) fn new(module: Arc<Module>, imports: &dyn ImportResolver) -> Result<Box<Instance>, String> {
+    // TODO visibility (in crate::runtime)
+    pub fn new(module: Arc<Module>, imports: &dyn ImportResolver) -> Result<Box<Instance>, String> {
```

Comment:
@lachlansneff  I reverted the visibility thing to work around a compile failure:
```
error[E0624]: method `new` is private
  --> src/bin/wasmer.rs:98:24
   |
98 |     let mut instance = wasmer::runtime::Instance::new(Arc::clone(&module), &import_object)
   |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error    
```